core/txn/pool/gatherer.go

```diff
@@ -1,75 +1,22 @@
 package pool
 
 import (
-	"bytes"
 	"context"
-	"sort"
-	"sync"
-
 	"go.dedis.ch/dela/core/access"
 	"go.dedis.ch/dela/core/txn"
 	"go.dedis.ch/dela/core/validation"
 	"golang.org/x/xerrors"
+	"sync"
+	"time"
```

Comment:
the current ordering of the imports seems a bit off.
could it be that you're not using goimports ?